monitoring_service.rst

```diff
@@ -2,145 +2,230 @@ Raiden Network Monitoring Service
+*   Sybil Attack resistance (i.e. no one should be able to announce an unlimited number of possibly faulty services)
+*   Some degree of redundancy (ability to register a balance proof with multiple competing monitoring services)
 
+Design of the Monitoring Service
+================================
 
-General requirements
---------------------
+Monitoring Service Registration
+-------------------------------
 
-MS that wish to get assigned a MS address (term?) in the global chat room MUST provide a registration deposit via SC [TBD]
+The Monitoring Service is part of the Raiden Service Bundle (RSB). RSB slots, the set of Raiden Service Providers, will
+be auctioned. If chosen in the auction, the service provider will become part of the list of MSs and must deposit some
+RDN.
 
-Users wishing to use a MS are RECOMMENDED to provide a reward deposit via smart contract [TBD]
+Client Onboarding
+-----------------
 
-Users that want channels to be monitored MUST post BPs concerning those channels to the global chat room along with the reward amount they’re willing to pay for the specific channel. They also MUST provide proof of a deposit equal to or exceeding the advertised reward amount. The offered reward amount MAY be zero.
+Clients that want to request this service have to deposit an amount of RDN into the UDC in order to provide rewards to
+the MS. This happens during the general Raiden onboarding process, so that no additional preparation is necessary when
+usage of a MS is desired.
 
-Monitoring services MUST listen in the provided global chat room
+Service Discovery
+-----------------
 
-They can decide to accept any balance proofs that are submitted to the chat room.
+All MS listen to a public Matrix room. Monitor Requests are broadcast and no specific MSs are appointed. The MSs can
+also publish their expected rewards in this room, which does not provide any guarantees, but increases the chance of
+reliable monitoring if both parties cooperate.
 
-Once it does accept a BP it MUST provide monitoring for the associated channel at least until a newer BP is provided or the channel is settled. MS SHOULD continue to accept newer balance proofs for the same channel.
+Monitoring Service Payment
+--------------------------
 
-Once a `ChannelClosed` or `NonClosingBalanceProofUpdated` event is seen the MS MUST verify that the channel’s balance matches the latest BP it accepted. If the balances do not match the MS MUST submit that BP to the channel’s `updateNonClosingBalanceProof` method.
+The MS is paid after successfully submitting its client’s balance proof update. The payment is paid out from a deposit
+in the User Deposit Contract (UDC). Ideally, only one MS submits the latest BP to the SC to avoid unnecessary gas usage.
```

Comment:
Can you add a note that this is still WIP

Reply:
yes